controllers/util/solr_util.go

```diff
@@ -101,7 +101,7 @@ func GenerateStatefulSet(solrCloud *solr.SolrCloud, solrCloudStatus *solr.SolrCl
 	defaultMode := int32(420)
 
 	probeScheme := corev1.URISchemeHTTP
-	if solrCloud.Spec.SolrTLS != nil {
+	if solrCloud.UrlScheme() == "https" {
```

Comment:
this is going to conflict with the Ingress TLS Termination PR. When it does, just change this to `SolrCloud.UrlScheme(false)` (getting url scheme internal in the cluster)

Reply:
this is an oops actually, sorry 'bout that (some cruft from an earlier approach), will revert back to what was there